josm-presets/de-signals-bostrab.xml

```diff
@@ -576,6 +576,38 @@ See https://wiki.openstreetmap.org/wiki/OpenRailwayMap for details.
 			</item>
 		</group>
 		<group name="Electricity Signals (St)" de.name="Schaltsignale (St)">
+			<item name="Signalcontact (St 1)" de.name="Signalkontakt (St 1)" type="node">
+				<label text="Signalcontact (St 3)" de.text="Signalkontakt (St 1)" />
```

Comment:
I am pretty sure that "Signalcontact" even correctly spelled as "signal contact" is not the correct translation but I do not know a better one. Maybe you ask on the OpenRailwayMap mailing list by describing what the signal does? I found "signal plunger" as a translation of "Signalkontakt" in an [English translation of BOStrab by G. H. Hafter](https://web.archive.org/web/20160615014031if_/http://www.modernstreetcar.org:80/pdf/bostrab-en_version_2008-05-12.pdf) but I have the feeling that a native English speaker familiar with interlocking will not understand it (however I failed to understand the pure BOStrab description of the signal as well ;-) ).

Reply:
In Hanover St 1 is often used to request a signal at a main signal if it is not coming automatically. So maybe we can call it "Signal requester"? I'm not sure how St 1 is used in another regions.